examples/test_api.c

```diff
@@ -118,35 +119,35 @@ int test_abtoull(char* str, unsigned long long* val)
   unsigned long long units = 1;
   if (*next != '\0') {
     switch(*next) {
-    case 'k':
-    case 'K':
-      units = kilo;
-      break;
-    case 'm':
-    case 'M':
-      units = mega;
-      break;
-    case 'g':
-    case 'G':
-      units = giga;
-      break;
-    case 't':
-    case 'T':
-      units = tera;
-      break;
-    case 'p':
-    case 'P':
-      units = peta;
-      break;
-    case 'e':
-    case 'E':
-      units = exa;
-      break;
-    default:
+      case 'k':
```

Comment:
Generally speaking, I think it is best to my stylistic updates to source code as a separate pass through the use of a tool so that the style may be made consistently across the entire file and all of the files in the project.  I suggest we undo the style updates so that we are left with just the changes required for the new functionality.

Reply:
I agree. This was the clang formatted screwing me up.